client/containers/Pub/PubSubmissionHeader/spubHeader.scss

```diff
@@ -0,0 +1,43 @@
+.spub-header-component {
+	border-bottom:	border-bottom: 1px solid #d2d2d2;
+;
+	padding: 200px;
+
+	h1 {
+		margin: 0;
+
+		font-size: 40px;
+		font-style: normal;
+		font-weight: 400;
+		line-height: 56px;
+		letter-spacing: 0em;
+		text-align: left;
+
+		box-decoration-break: clone;
+	}
+
+	.tabs{
+		color: var(--community-accent-dark);
+		max-width: 900px;
+		> .bp3-tab-list {
+			position: absolute;
+			width: 1005.79px;
+			height: 0px;
+			padding-top: 5px;
+			.tab {
+				color: var(--community-accent-dark);
+				font-size: 14px;
+				font-style: normal;
+				font-weight: 500;
+				line-height: 17px;
+				letter-spacing: 0.15em;
+				text-align: left;
+				padding: 15px;
+				text-transform: uppercase;
+			}
+		}
+		.tab-panel {
+			padding-top: 33px;
```

Comment:
After removing the `position: absolute`, something happens where this looks like it may be too much padding in Storybook.